pennylane/ops/op_math/controlled.py

```diff
@@ -597,18 +597,6 @@ def __init__(
     def _inverse(self):
         return False
```

Comment:
I think this can go too, since `Operator._inverse` should now always be False (from its `__init__`)